### ISSUE TITLE
Migrate Firefox for Android (Nightly) to Pontoon

### ISSUE THREAD

**Issue by gueroJeff** [mozilla-l10n/pm-projects#100]
Given the information about the next deadline for this project, it is the assignee's decision when to schedule the migration for this project so that it does not add risk to its l10n delivery.

Next merge day: 20 September
## Shipping locales

- [x] ach
- [x] an
- [x] ar
- [x] as
- [x] bn-BD
- [x] br
- [x] bs
- [x] ca
- [x] cak
- [x] cy
- [x] dsb
- [ ] en-ZA
- [x] eu
- [x] fa
- [x] ff
- [x] ga-IE
- [x] gd
- [x] gn
- [x] hi-IN
- [x] hr
- [x] hsb
- [x] hu
- [x] hy-AM
- [x] is
- [x] kk
- [x] kn
- [x] lt
- [x] mai
- [x] ml
- [x] mr
- [x] ne-NP
- [x] or
- [x] pt-PT
- [x] ro
- [x] son
- [x] ta
- [x] ur
- [x] uz
- [x] xh
- [x] zam

## Close-to-shipping locales

- [ ] hto
- [x] meh
- [x] mix
- [x] pai
- [ ] pbb
- [x] quy
- [ ] qvi
- [x] trs

## Others

- [ ] af
- [ ] ak
- [ ] am
- [ ] anp
- [ ] arn
- [ ] bcl
- [ ] bm
- [ ] bo
- [ ] cbk
- [ ] ckb
- [ ] cly
- [ ] co
- [ ] crh
- [ ] ctu
- [ ] doi
- [ ] dz
- [ ] en-IE
- [ ] fj
- [ ] fo
- [ ] frp
- [ ] fur
- [ ] got
- [ ] gv
- [ ] haw
- [ ] hch
- [ ] ht
- [ ] hto
- [ ] hus
- [ ] ilo
- [ ] ixl
- [ ] jbo
- [ ] jv
- [ ] kok
- [ ] krc
- [ ] ku
- [ ] kw
- [ ] ky
- [ ] lb
- [ ] lg
- [ ] mg
- [ ] mn
- [ ] mni
- [ ] mqh
- [ ] mxp
- [ ] nch
- [ ] ncj
- [ ] nd
- [ ] neb
- [ ] noa
- [ ] nqo
- [ ] nso
- [ ] om
- [ ] pms
- [ ] prs
- [ ] ps
- [ ] quz
- [ ] roa-ES-val
- [ ] rw
- [ ] sa
- [ ] sah
- [ ] sat
- [ ] sc
- [ ] sco
- [ ] sd
- [ ] shn
- [ ] sm
- [ ] ss
- [ ] su
- [ ] sw
- [ ] szl
- [ ] tg
- [ ] ti
- [ ] tk
- [ ] tn
- [ ] toj
- [ ] trc
- [ ] tsz
- [ ] tt-RU
- [ ] tzh
- [ ] tzo
- [ ] ug
- [ ] wa
- [ ] yaq
- [ ] yo
- [ ] yua
- [ ] zu

**Comment by Delphine** [comment]
Decided on August 21st. 
Working on email to Android localizers now - those I haven't already reached out today in my mobile outreach will get their individual email tomorrow

**Comment by flodolo** [comment]
A couple of notes:
* If the locale doesn't have a Mercurial repository on hg.mozilla.org, you need me to set up the repository on BitBucket.
* If it's not already enabled, you need to enable Firefox too (for shared parts).

**Comment by Delphine** [comment]
The only 2 locales that don't have Mercurial repos and that I plan to move are "pai" and "qvi". Thanks for your help flod!

**Comment by flodolo** [comment]
> The only 2 locales that don't have Mercurial repos and that I plan to move are "pai" and "qvi". Thanks for your help flod!

OK. Make sure to create the locales on Pontoon, I'll enable Firefox and Fennec tomorrow morning after creating and populating the repos.

**Comment by flodolo** [comment]
Repositories are created for pai, quy. In case, you only need to create the locale and enable the projects
https://bitbucket.org/mozilla-l10n/

**Comment by Delphine** [comment]
I've double-checked and all Fennec shipping locales have their Firefox desktop counterpart. 
The locales I've disabled from Pootle but not enabled under Pontoon are:
   hto
   pai
   pbb
   quy
   qvi
because I need to know how their plural forms work before I can open up a page for them in Pontoon. I've reached out to Monica about this, since I don't know who the localizers for these projects are at this point (no team page).

I'll close this issue once we've enabled all these missing locales to Pontoon.
